pulsar-broker/src/main/java/org/apache/pulsar/broker/service/BrokerService.java

```diff
@@ -2314,7 +2314,8 @@ private void updateConfigurationAndRegisterListeners() {
         }
 
         // add listener to notify topic subscriptionTypesEnabled changed.
-        registerConfigurationListener("subscriptionTypesEnabled", this::updateBrokerSubscriptionTypesEnabled);
+        registerConfigurationListener("subscriptionTypesEnabled",
```

Comment:
Personally, I am in favor of the previous version which save one anonymous lambda.